lib/amqp/client/channel.rb

```diff
@@ -185,9 +191,9 @@ def basic_reject(delivery_tag, requeue: false)
       write_bytes FrameBytes.basic_reject(@id, delivery_tag, requeue)
     end
 
-    def basic_qos(prefetch_count, prefetch_size: 0, global: false)
-      write_bytes FrameBytes.basic_qos(@id, prefetch_size, prefetch_count, global)
-      expect :basic_qos_ok
+    def basic_recover
```

Comment:
`requeue` should be an argument here, that's passed to `FrameBytes.basic_recover`, no?